README.md

```diff
@@ -18,6 +18,10 @@ functions:
     name: ${self:provider.stage}-${self:service}-process-event-batch
     description: ${git:branch} - ${git:describe} - ${git:sha1}
 
+  processEventBatch2:
```

Comment:
Since describeLight isn't a built-in for git, can you add some more documentation to the README as to what it does?

Reply:
Sure thing :)

Reply:
Commit `#44a193e` has added documentation as requested.